### DISCUSSION TITLE
Aosp 10 v220

### DISCUSSION THREAD

**Discussion by jimmy34556** [phhusson/treble_experimentations#1882] [Q&A]
I successfully flashed Android 10 to my device, but my internal storage is gone , from 64 gb to 4 gb,,, I also can't receive messages, and the messages I sent didn't go through,please help
![Screenshot_20210604-081550](https://user-images.githubusercontent.com/85335346/120761515-2f13b700-c50d-11eb-9f68-ed18b548a6fc.png)

**Comment by phhusson** [comment]
Do factory reset from stock unmodified recovery.

**Reply by jimmy34556** [reply]
Thanks,it worked,,,,but I still can't receive messages

**Reply by sebastianjoh** [reply]
how to use the patch.zip`s i can`t flash the patch`s zip in s6 edge

**Comment by phhusson** [comment]
Well that's another issue, you're not supposed to have multiple issues in
one issue...

Anyway, I'm in good mood, let's say I allow it.

Do you have the issue on both SIMs? Do you use Google Messages or another
SMS app? What are your carriers?

**Reply by jimmy34556** [reply]
I have the issues on both Sims , it's not working on Google messages and chomp sms ,and my carriers are Airtel and MTN

**Comment by sebastianjoh** [comment]
hello any body here can explain how to use patch zip`s here
